python/cugraph/opg/link_analysis/mg_pagerank_wrapper.pyx

```diff
@@ -39,15 +39,14 @@ def mg_pagerank(input_df, local_data, handle, alpha=0.85, max_iter=100, tol=1.0e
     local_offset = dst.min() # Find using local data offset array
```

Comment:
Perhaps size info should be set in the python graph directly only once and just accessed afterward.

Reply:
updated